### PR TITLE
[IMP] portal: categories & common cards 

### PR DESCRIPTION
**Pull request by fdamhaut**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by chgo-odoo** [comment]
Hi @Brieuc-brd 👋

Could you have a look at this PR and the Enterprise counterpart? https://github.com/odoo/enterprise/pull/45574

Thanks !

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
**This review is only for 308a20b8a275902fd446b167174ba6cda27b45a0**

Thanks for your work @chgo-odoo @morm-odoo, I’m sure this wasn’t easy and this commit redesign is pretty huuuge, and it was not easy to review 😅 . Here is a first review, I might go further once the code is cleaner :) 

BTW, it would be easier to review if you could split by each apps for example.

### Duplicated content :

I’ve seen a lot of duplicated content in each apps, do you think it’s possible to simplify this by creating common templates in `portal` and use them in the different apps ?

### Attachment files :

The design feels outdated. Maybe we could do something like the card we have for Discuss ?
| Portal | Discuss |
|--------|--------|
|Capture d’écran 2023-09-29 à 09 07 09 |Capture d’écran 2023-09-29 à 15 57 41  |

### Tasks :

- Is it necessary to call the task name here ?
Capture d’écran 2023-09-29 à 09 47 16

### Addresses Folders :

Maybe I'm getting old 👴 , but it took me a while to figure out how to change my profile.
For me, the "Addresses" folder isn't clear enough to know that this is also where you can change your name and personal information. What do you think?
Capture d’écran 2023-09-29 à 16 01 23

### Could you also double check : 

- If the classes you set are really necessary. I found a lot of iterations where they were not.
- make sure the classes are in the right order.

Don’t hesitate if you need help :)

Courage ! 💪

cc. @stefanorigano

File: addons/portal/views/portal_templates.xml [hunk 1]
```diff
@@ -126,36 +126,66 @@
...
+                            <button class="btn position-relative d-lg-none align-self-center border-0 bg-transparent"
+                            data-bs-toggle="offcanvas"
+                            data-bs-target="#accountOffCanvas">
+                                <img class="rounded-circle"
+                                t-att-src="image_data_uri(user_id.partner_id.avatar_1024)" alt="Contact" width="24"/>
+                                <span t-out="user_id.name"/>
+                            </button>
+                        </div>
+                        <div class="row justify-content-between">
+                            <div t-attf-class="col-12 col-lg-9 border-0">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/portal/views/portal_templates.xml:149]: To make sure it doesn't inherit the border-left property defined in `portal.scss`

File: addons/portal/views/portal_templates.xml [hunk 2]
```diff
@@ -192,49 +222,59 @@
...
-                <span class="badge text-bg-secondary rounded-pill" t-esc="count"/>
-            </t>
-            <t t-elif="placeholder_count">
-                <span class="badge text-bg-secondary rounded-pill" t-att-data-placeholder_count="placeholder_count">
-                    <i class="fa fa-spin fa-circle-o-notch"></i>
-                </span>
-            </t>
-        </a>
+        <div t-att-class="('' if config_card else 'd-none ') + ('col-12 order-0' if show_count else 'col-md-6 order-2')" id="portal_card">
+            <a t-att-href="url" t-att-title="title" t-attf-class="d-flex justify-content-start gap-3 align-items-center h-100 #{bg_color if bg_color else 'bg-light p-3'} text-decoration-none">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/portal/views/portal_templates.xml:256]: `text-decoration-none` class is needed to override the hover behavior

File: addons/portal/views/portal_templates.xml [hunk 3]
```diff
@@ -263,7 +303,7 @@
         :param classes : navbar classes
     -->
     <template id="portal_searchbar" name="Portal Search Bar">
-        <nav t-attf-class="navbar navbar-light navbar-expand-lg border py-0 mb-2 o_portal_navbar {{classes if classes else ''}} {{'mt-3 rounded' if breadcrumbs_searchbar else 'border-top-0' }}">
+        <nav t-attf-class="navbar navbar-expand-lg mb-2 p-0 o_portal_navbar {{classes if classes else ''}} {{'mt-3 rounded' if breadcrumbs_searchbar else 'border-top-0' }}">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/portal/views/portal_templates.xml:326]: double check à partir d'ici

File: addons/portal/views/portal_templates.xml [hunk 4]
```diff
@@ -276,12 +316,12 @@
             </button>
 
             <!--  Collapsable content  -->
-            <div class="collapse navbar-collapse" id="o_portal_navbar_content">
-                <div class="nav flex-column flex-lg-row ms-auto p-0 mb-3 mb-lg-0 mt-1 mt-lg-0">
+            <div t-attf-class="collapse navbar-collapse align-items-stretch justify-content-start justify-content-md-end flex-wrap flex-lg-nowrap border-top-0 bg-transparent {{'flex-wrap' if breadcrumbs_searchbar  else ''}}" id="o_portal_navbar_content">
```
[the remark this answers is not in the document]
Comment (chgo-odoo) [reply addons/portal/views/portal_templates.xml:339]: double check ici

**Comment by fdamhaut** [comment]
Merged in #138077

**Pull request closed without merging**